OracleTuxedo/tuxedo_full/README.md

```diff
@@ -0,0 +1,16 @@
+
+Tuxedo + SALT + TSAM Plus agent on Docker
+===============
+
+## How to run
+#Before running buildDockerImage.sh, you need to set the environment variables, http_proxy, https_proxy, ftp_proxy, and no_proxy, to access internet if the docker environment is behind a corporate proxy.
+
+The base image oracle/tuxedo:12.2.2 should be built before you run buildDockerImage.sh.
+Before getting started, download the Tuxedo 12.2.2 Linux 64 bit installer from http://www.oracle.com/technetwork/middleware/tuxedo/downloads/index.html
```

Comment:
Please provide the exact filename that needs to be downloaded.

Reply:
added filename